definitions/ext-flex_host/definition.yml

```diff
@@ -0,0 +1,30 @@
+domain: EXT
+type: FLEX_HOST
+synthesis:
+  name: displayName
+  identifier: displayName
```

Comment:
Is there any limits on the displayName attribute? if not we may want to add `      encodeIdentifierInGUID: true`

Reply:
no limits, have added this now thx